native/java/org/jpype/ref/JPypeReference.java

```diff
@@ -1,32 +1,25 @@
-package jpype.ref;
+package org.jpype.ref;
 
 import java.lang.ref.PhantomReference;
 import java.lang.ref.ReferenceQueue;
 
 public class JPypeReference extends PhantomReference
 {
-  private long mHostReference;
+  long mHostReference;
```

Comment:
why change visibility here?

Reply:
It was requiring multiple accessor to change something which was just a package level variable.   Thus to save space I trimmed out anything which should not be seen on the outside.  The entire JPypeReference class is likely to be changed into a private internal class.